src/0_fsd_common/fsd_common_msgs/msg/CarState.msg

```diff
@@ -1,3 +1,4 @@
 std_msgs/Header header
 
-geometry_msgs/Pose2D car_state    # Position in x, y, theta
\ No newline at end of file
+geometry_msgs/Pose2D car_state    			# Position in x, y, theta
+fsd_common_msgs/CarStateDt car_state_dt		# Velocities
```

Comment:
Now we have CarStateDt which also has a header in CarState... hmm thinking how we can make the messages better

From VE we need just the velocities, then from there on I guess position and velocity is always coupled (estimation, mapping, control), right? So we could have CarState to have both position and velocity, and then a separate stamped only-velocity message type? 

Guess it's not thaaattt ugly, but it bothers me :)

Reply:
you need to have them combined since you want them synced. Well, you can always distinguish them with the `Stamped` name